src/components/App/App.js

```diff
@@ -42,10 +45,50 @@ const App = () => {
             render={() => {
               return (
                 <>
-                  <h2>Take the Sexual Health Myth Debunking Quiz!</h2>
-                  <Link to="/quiz">YES!</Link>
+                  <div className="quiz-call-to-action">
+                    <h2 className="call-to-action-question">
+                      Does wearing two condoms offer double protection?
+                    </h2>
+                    <Link to="/quiz" className="quiz-link" onClick={ resetScore }>
+                      Take the Myth Busting Quiz
+                      <span class="material-icons">
+                        arrow_right_alt
+                      </span>
+                    </Link>
+                  </div>
+                  <div className="fact-container">
+                    <h2 className="did-you-know">Did you know? 🤔</h2>
+                    <p className="fact">"{questions.length && questions[step].fact }"</p>
+                    <div className="arrows-container">
+                      <button
+                        className="arrow-button"
+                        disabled={ step ? false : true }
+                        onClick={() => setStep(step - 1)}
+                      >
+                        <span class="material-icons">
+                          arrow_back
+                        </span>
+                      </button>
+                      <button
+                        disabled={ step < questions.length - 1 ? false : true }
+                        className="arrow-button"
+                        onClick={() => setStep(step + 1)}
+                      >
+                        <span class="material-icons">
+                          arrow_forward
+                        </span>
+                      </button>
+                    </div>
+                  </div>
+                  {
```

Comment:
I will remove this form for now, I am not sure how to make this functional with my current API.